ci/check_style.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
```

Comment:
Probably a holdover from an earlier iteration? There are no changes in this file other than the copyright.

Reply:
Thanks, I reverted that in the latest commit.